tests/test_dataset_generator.py

```diff
@@ -0,0 +1,161 @@
+"""Contains tests for dataset_generator"""
+
+import unittest
+from collections import OrderedDict
+
+import numpy as np
+import pandas as pd
+
+from synthetic_data import dataset_generator as dg
+
+
+class TestDatasetGenerator(unittest.TestCase):
+    def test_get_ordered_column_datetime(self):
```

Comment:
tests that get_ordered_column returns datetime numpy array in ascending order while still maintaining the original datetime format inserted into it